plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/IStreamValueEditor.java

```diff
@@ -67,4 +70,7 @@ void contributeSettings(@NotNull IContributionManager manager, @NotNull CONTROL
 
     void disposeEditor();
 
+    @Nullable
+    File getExternalFile(@NotNull CONTROL control);
```

Comment:
Let's move it to a separate interface (e.g. IStreamValueEditorPersistent) and implement it only in browser. 
And use nio Path instead of File.